pystac_client/cli.py

```diff
@@ -133,6 +134,7 @@ def cli():
     # don't enable logging if print to stdout
     if args.get('save', False) or args.get('matched', False):
         logging.basicConfig(stream=sys.stdout, level=loglevel)
+        # coloredlogs.install(level=loglevel, logger=logger)
```

Comment:
Do you want this comment in here?

Reply:
will remove, was trying out colored terminal logs during development